src/gl/directx11/shaders/fenceFragmentShader.hlsl

```diff
@@ -36,6 +36,5 @@ PS_OUTPUT main(PS_INPUT input)
   float halfFcoef = 1.0 / log2(s_CameraFarPlane + 1.0);
   output.Depth0 = log2(input.fLogDepth.x) * halfFcoef;
 
-  output.Color0.a = output.Depth0; // depth packed here
```

Comment:
I don't think removing these is correct- this takes as a step further from fences being selectable in the scene? @peterAdams1014 ?